lib/safari-web/index.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const Nightmare = require('nightmare');
+const logger = require('../logger');
+
+Nightmare.action(
+  'flushCookies',
+  (name, options, parent, win, renderer, done) => {
+    parent.respondTo('flushCookies', done => {
+      win.webContents.session.cookies.flushStore(done)
+    })
+    done()
+  },
+  function(done) {
+    this.child.call('flushCookies', done)
+  }
+)
+
+const SafariWeb = function SafariWeb(browserDataDir) {
+  this.nightmare = Nightmare({
+    show: true,
+    waitTimeout: 180 * 1000,
+    pollInterval: 5000,
+    webPreferences: {partition: null},
+    paths: {userData: browserDataDir},
+    openDevTools: {mode: 'detach'}
+  })
+}
+
+SafariWeb.prototype.login = function login() {
+  return new Promise(function(resolve) {
+    this.nightmare
+    .on('console', (log, msg) => {
+      logger.log(msg)
+    })
+    .goto('https://safaribooksonline.com')
```

Comment:
This needs to change to `https://learning.oreilly.com` for it to work

Reply:
Even with this change it did not work for me